app/controllers/invitations_controller.rb

```diff
@@ -1,28 +1,50 @@
 class InvitationsController < ApplicationController
   def create
-    invitation = Invitation.new(user_id: params[:user], friend_id: params[:friend], confirmed: false)
-    if invitation.save
-      redirect_to User.find(invitation.friend_id),
+    invitations = [Invitation.new(user_id: params[:user],
+                                  friend_id: params[:friend],
+                                  confirmed: false,
+                                  inviter_id: params[:user]),
+                   Invitation.new(user_id: params[:friend],
+                                  friend_id: params[:user],
+                                  confirmed: false,
+                                  inviter_id: params[:user])]
```

Comment:
#### NICE TO HAVE
- [ ] Please consider passing parameters to your controllers in a safe way using the `require` and `permit` methods. See this [Odin Project article](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-on-rails/lessons/controllers) for reference.

> I notice you're creating an additional row with reversed user-friend pair, please take a look at the summary for suggestions on that.